draft-ietf-quic-manageability.md

```diff
@@ -677,19 +677,26 @@ secret is retained by the observer.
 
 ## Flow Association {#sec-flow-association}
 
-The QUIC connection ID (see {{rebinding}}) is designed to allow an on-path
-device such as a load-balancer to associate two flows as identified by
-five-tuple when the address and port of one of the endpoints changes; e.g. due
-to NAT rebinding or address migration. An observer keeping flow state
-can associate a connection ID, if present, with a given flow, and can associate
-a known flow with a new flow when observing a packet sharing the same
-connection ID in the same direction between client and server and sharing one
-endpoint address (IP address and port) with the known flow.
-
-However, since the connection ID may change multiple times during the lifetime
-of a flow, and the negotiation of connection ID changes is encrypted, packets
-with the same 5-tuple but different connection IDs might or might not belong to
-the same connection.
+The QUIC connection ID (see {{rebinding}}) is designed to allow a coordinating
+on-path device, such as a load-balancer, to associate two flows when one of the
+endpoints changes address or port.  This change can be due to NAT rebinding or
+address migration.
+
+The connection ID must change upon intentional address change by an endpoint,
+and connection ID negotiation is encrypted, so it is only possible for a
+passive observer to link unintended changes of address using the connection ID.
+
```

Comment:
```suggestion
The connection ID must change upon intentional address change by an endpoint,
and connection ID negotiation is encrypted.
```
I'd prefer to keep the sentence below but remove this half sentence and make one paragraph out of it.

Reply:
```suggestion
The connection ID must change upon intentional address change by an endpoint,
and connection ID negotiation is encrypted, packets with the same 5-tuple but
different connection IDs might or might not belong to  the same connection.
```
Actually I would prefer to keep the original sentence here...